cypress/e2e/main.env.cy.ts

```diff
@@ -1,6 +1,6 @@
 const visualRegressionConfig = Cypress.env('visualRegression')
 visualRegressionConfig.baseDirectory = './cypress/snapshots/alternative-base'
-visualRegressionConfig.diffDirectory = './cypress/snapshots/alternative-diff'
+const screenshotsFolder = Cypress.config('screenshotsFolder')
```

Comment:
This was the bug.
We were checking against dif folder and we should check against base dir (snapshots/actual)

I guess we will also need to do some other checks to see if we are generating the diff folder

Reply:
I'm not sure that's ok 🤔 What we should test here is that changing where base and diff images are generated is properly storing files or not.
If you just use screenshotFolder, you check if the screenshot is there or not, but that's just regular cypress, nothing to do with this plugin

Reply:
But now I see that we are missing that 'generateDiff' flag is set to 'always'. Wth it worked locally for me 🤔 I'll check again

Reply:
Ok, then I see that the problem is about generating the diff images.

Looking at the code I see a couple of use cases,  and basically, what I realize is that we are not generating the diff images if the images are the same, that is why we are never going to get those images under the diff folder

looking at the image:

![image](https://github.com/Zaista/cypress-visual-regression/assets/2030605/0e8a8ea2-9588-4e50-bc82-3bd1e5c7a416)

It looks to me that we need to  setup `generateDiff === 'always'`

Reply:
> But now I see that we are missing that 'generateDiff' flag is set to 'always'. Wth it worked locally for me 🤔 I'll check again

I don't see that you are setting that flag anyware in the code 😅

Reply:
It was probably working locally for you because you didn't clean that folder and probably you made some old test and generate those files then. 

I added `visualRegressionConfig.generateDiff = 'always'` to the test and it looks that is passing now 🤣